packages/core/core/src/Transformation.js

```diff
@@ -146,7 +146,7 @@ export default class Transformation {
       // If no existing sourcemap was found, initialize asset.sourceContent
       // with the original contents. This will be used when the transformer
       // calls setMap to ensure the source content is in the sourcemap.
-      asset.sourceContent = await asset.getCode();
+      asset.sourceContent = await asset.getBuffer();
```

Comment:
Does this mean that assets with (probably large) stream content will be read into memory at once and stored on the asset cache entry?